src/app/utils/getNews.ts

```diff
@@ -0,0 +1,31 @@
+import { schemaNewsResponse } from '@/models/schemes';
+import { NewsResponse, NewsWithId } from '@/models/types';
+import axios from 'axios';
+import { z } from 'zod';
+
+const getNews = async (keyword: string) => {
```

Comment:
Please create a type for what this function will return. Make status and message literal types